hugo/data/you_draw_it_karlsruhe.json

```diff
@@ -0,0 +1,193 @@
+{
+  "chart_id": "you-draw-it-paris",
+  "start_year_chart": 2007,
+  "end_year_chart": 2050,
+  "start_year_draw": 2019,
```

Comment:
In Karlsruhe wurde die aktuellste CO2 Bilanz bisher lediglich für das Jahr 2015 veröffentlicht. Die Benutzerin sollte also im Jahr 2016 anfangen zu zeichnen.

Reply:
So wie ich den you-draw-it Chat verstehe, soll er eine intuitive Mitmach-Einführung in das Thema geben. Dadurch, dass die neue Version auch "nur" 5-Jahres-Schritte darstellt, haben wir ohnehin ein gewisses Genauigkeitsproblem (da die Daten selten in diesen 5 Jahres-Schritten vorliegen). Die Idee ist also, dass der selber-zeichnen-Plot recht grob ist (aber natürlich nicht falsch) und die Detailinformationen im nächsten Plot erkundbar sind (also: was sind tatsächliche Daten, was Hochrechnung.) Tatsächlich ist es so, dass für die neue you-draw-it Version alle 5 Jahre ein Datenpunkt eingezeichnet wird -- wenn er vorhanden ist -- und wenn er nicht vorhanden ist wird der Wert des linearen Trends eingezeichnet.

Darüber hinaus geht die Berechnung des Paris-Budgets (was der/die NutzerIn selber einzeichnen soll) vom Jahr 2020 aus (siehe hier: https://github.com/codeformuenster/klimawatch/blob/master/generate_plots.py#L101). Dazu wird angenommen, dass 2019 die Emissionen so hoch waren, wie die letzten offiziellen Daten. (Vielleicht wäre auch hier der Trend eine bessere Annahme?) Hauptgrund: Das deutschlandweite CO2-Budget aus dem Blogartikel von Rahmstorf ist für Anfang 2019 berechnet und das Skript zieht dann noch ein Jahr Emissionen ab.
Für eine intuitive Einschätzung der benötigten _zukünftigen_ CO2-Emissionen fände ich ein Einzeichnen von 2016 aus auch etwas komisch, da das ja die _Vergangenheit_ betrifft.

Insgesamt sollten wir aber vielleicht nochmal genereller über diese Dinge nachdenken und diskutieren. Aber evtl. besser in einem Issue? Oder im Chat?

Reply:
Im Sinne der Transparenz und Nachvollziehbarkeit (und somit auch der Belastbarkeit der Graphen) sollten diese Details
a) direkt unter den Digrammen stehen (evtl. reveal per Klick)
b) in einfacher Sprache gehalten sein
c) evtl. in einem Issue diskutierbar sein

Reply:
Die CO2 Bilanz einer Kommune wird auf absehbare Zeit um ca. ein bis zwei Jahre dem aktuellen Jahr hinterherhinken. Dies liegt daran, wie die notwendigen Input-Daten reportet bzw. veröffentlicht werden (statistische Landes-, Bundesämter). Ich finde das nicht unbedingt gut, aber so ist es meines Wissens nach die letzten Jahre gewesen.

Daher sollte:
a) der Datenpunkt der letzten veröffentlichten Bilanz für jede Kurve klar gekennzeichnet sein
b) danach ein Trendwert verwendet werden (evtl. andere Farbe)
c) der aktuelle Zeitpunkt (aktuelles Jahr) im Graph markiert sein
d) im you-draw-it immer ab dem aktuellen Zeitpunkt gezeichnet werden